torch/_tensor.py

```diff
@@ -53,7 +65,10 @@ def _rebuild_from_type_v2(func, new_type, args, state):
     # Tensor does define __setstate__ even though it doesn't define
     # __getstate__. So only use __setstate__ if it is NOT the one defined
     # on Tensor
-    if getattr(ret.__class__, "__setstate__", Tensor.__setstate__) is not Tensor.__setstate__:
+    if (
+        getattr(ret.__class__, "__setstate__", Tensor.__setstate__)
+        is not Tensor.__setstate__
+    ):
```

Comment:
This seems much worse than before for me.

Reply:
Well, AFAIK this is the same kind of setup used internally by fbsource. So I opt for this by default for consistency. Let me see if we can tweak this